examples/ecommerce-netlify-functions/models.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+const mongoose = require('mongoose');
+
+const productSchema = new mongoose.Schema({
+  productName: String,
+  productPrice: Number
+});
+
+const Product = mongoose.model('Product', productSchema);
+
+module.exports.Product = Product;
+
+const orderSchema = new mongoose.Schema({
+  items: [{ productId: { type: mongoose.ObjectId, required: true, ref: 'Product' }, quantity: { type: Number, required: true } }],
```

Comment:
Break it up so `productId` and `quantity` are on separate lines. And add validation to `quantity` : `validate: v => v > 0`